eng/targetingpacks.targets

```diff
@@ -1,4 +1,8 @@
 <Project>
+  <PropertyGroup>
+    <SharedFrameworkName Condition="'$(SharedFrameworkName)' == ''">Microsoft.NETCore.App</SharedFrameworkName>
```

Comment:
(minor side question): When/why is this necessary?

Reply:
I'm intending to use this file for the linker infrastructure to avoid the duplication of the runtime pack bootstrapping logic. As I was already at it, I defaulted the property so that there's one less that needs to passed in globally when building the linker tests.